model/get_users_struct.go

```diff
@@ -0,0 +1,6 @@
+package model
```

Comment:
Also I think the struct name in file  name is unnecessary as it's a model which tells that it will be some model and they are struct

Reply:
Fixed.